src/App.jsx

```diff
@@ -14,33 +16,143 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+export const App = () => {
+  const [state, setState] = useState({
+    sortedGoods: [...goodsFromServer],
+    buttonSortAplha: { 'is-light': true, 'button is-info': true },
+    buttonSortLength: {
+      'is-light': true,
+      'button is-info': true,
+      'is-success': true,
+    },
+    buttonReverse: { button: true, 'is-warning': true, 'is-light': true },
+    buttonRestart: false,
+  });
```

Comment:
You have several states that, although related to each other, are not exactly the same; you should use a `React.useState` for each of them (or a `React.useReducer` if you really need this logic)